### PR TITLE
Conditions are not used anymore in PFEM fluid

### PR DESCRIPTION
**Pull request by AFranci**

### CODE REVIEW

File: applications/DelaunayMeshingApplication/custom_processes/mesher_process.hpp [hunk 1]
```diff
@@ -79,49 +79,49 @@ class MesherProcess : public Process
 
     /// this function is designed for being called at the beginning of the computations
     /// right after reading the model and the groups
-    void ExecuteInitialize() final
+    void ExecuteInitialize() override
     {
```
Comment (josep-m-carbonell) [inline applications/DelaunayMeshingApplication/custom_processes/mesher_process.hpp]: These methods must be "final". Mesher processes can only call Execute(), other methods are not called. Therefore no override method is allowed.

**Review by josep-m-carbonell — CHANGES_REQUESTED** [review]

File: applications/DelaunayMeshingApplication/custom_processes/mesher_process.hpp (hunk 1 above)
Comment (AFranci) [reply applications/DelaunayMeshingApplication/custom_processes/mesher_process.hpp]: It was giving a warning suggesting to use override and not final. I will check it again

Comment (AFranci) [reply applications/DelaunayMeshingApplication/custom_processes/mesher_process.hpp]: screenshot from 2018-09-13 17-15-00

Comment (philbucher) [reply applications/DelaunayMeshingApplication/custom_processes/mesher_process.hpp]: I think it asks you to use both: `final override`

Comment (AFranci) [reply applications/DelaunayMeshingApplication/custom_processes/mesher_process.hpp]: Is this ok for you @josep-m-carbonell? with 'final override' the warnings do not appear...

Comment (josep-m-carbonell) [reply applications/DelaunayMeshingApplication/custom_processes/mesher_process.hpp]: Yes sure, I didn't get this warning from the compiler, using"final" was enough..but "final override" I guess it means the same. Thank you @philbucher and @AFranci.

**Review by josep-m-carbonell — APPROVED** [review]

**Pull request merged**